diff --git a/dist/index.js b/dist/index.js
index 12e32e0d33ab28c71a80a593730152f7d7909ea2..cc60af942c9df0881711f5ae470d7481bbe274c4 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -204,7 +204,7 @@ async function* events(res, signal) {
   let iter = stream(res.body);
   let line, reader = iter.getReader();
   let event;
-  for (; ; ) {
+  for (; ;) {
     if (signal && signal.aborted) {
       return reader.cancel();
     }
@@ -386,7 +386,7 @@ function processText(output) {
   if (output?.choices?.[0]?.message?.content?.length) {
     return output?.choices?.[0]?.message?.content;
   }
-  if ("response" in output) {
+  if (output && "response" in output) {
     if (typeof output.response === "object" && output.response !== null) {
       return JSON.stringify(output.response);
     }
